.github/workflows/validate_tests.yml

```
@@ -0,0 +1,24 @@
+name: Validate test data
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: '**'
+
+jobs:
+  run_all:
+    name: Validate tests using schema
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Install CLI tool for JSON Schema validation
+        run: npm install --global ajv-cli
+      - name: Validate tests using the latest schema version
+        run: |
+            ajv validate --spec=draft2020 \
+            -s $(ls -1v schemas/*/*schema.json | tail -1) \
+            -d 'tests/**/*.json'
```

With `|` block literals, newlines are literal and therefore the trailing `\` are needed so that the shell can combine these into a single command. With a `>` block literal, the newlines are collapsed into single spaces, and the `\` aren't required.

```suggestion
        run: >
            ajv validate --spec=draft2020
            -s $(ls -1v schemas/*/*schema.json | tail -1)
            -d 'tests/**/*.json'
```